frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/pipeline-metrics-utils.ts

```diff
@@ -10,6 +10,7 @@ import {
   dateFormatterNoYear,
   parsePrometheusDuration,
 } from '@console/internal/components/utils/datetime';
+import i18n from '@console/internal/i18n';
```

Comment:
There is already an import `import i18next from 'i18next';`, all other translation code here uses `i18next.t(`, could we reuse this here as well?